### ISSUE TITLE
User suggestion: PhET Tutorial/Documentation

### ISSUE THREAD

**Issue by oliver-phet** [phetsims/tasks#1043]
@ariel-phet requested this be tagged for developer meeting.
> Hi, I was asked to flesh out my request for PhET help resources.
> 
> I think what would be most helpful would be a tutorial built off a more complex sim, one that walks you through adding most of the major features. So it'd start off with the simplest minimal sim generated, explaining the major important components. Then, you'd do something like add a static image. Then you might make that image draggable or something. From there, perhaps adding a button or menu. You'd keep adding bits and pieces incrementally, explaining each step and what your options are. To be honest, it's a little difficult for me to describe what order to introduce topics, because I still don't have a handle on what the possibilities even are. I'd prefer this as a formatted web document, although I know some of my students prefer video tutorials (I don't know that you'd find consensus).
> 
> Which brings me to my second suggestion, a more cohesive collection of API documentation. I think even just running jsdoc over your codebase would be a reasonable start, but it'd be really nice to get a collection of features that break down what code is needed. For comparison, my students have been using Phaser to make educational games; the API documentation can be navigated from the top of this page, and although it's a bit of a laundry list it still allows you to get a sense of what options you have if you click around for a while. I was looking around for a similar resource for PhET but couldn't find anything.
> 
> Perhaps a minimum, if you were eager to get something thrown together, is to just record a video of someone developing a sim from the template, explaining what they are doing along the way. Even that would be pretty helpful as a starting point.
> 
> I hope this helps! PhET seems really cool, and long term I'd like to have my students use your platform instead of Phaser. However, I need the development process to be a little more accessible to novice Computer Scientists. It's a tall ask, so I understand this may not be your top priority, but hopefully something to work towards!

**Comment by jessegreenberg** [comment]
Discussion 10/8/20 - 

This would be really nice to have. But we also recognized that this could be a project of large scope. Comparing to other resources, we recognized that we would need to consider examples, how to present them, what it should look like, things involving thoughtful design.

It was also mentioned during meeting that we could improve our auto-generation of documentation for code with JSDoc.

@ariel-phet is being reassigned for now.
